app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Users::SessionsController < Devise::SessionsController
+  include RackSessionFix
+  respond_to :json
+
+  private
+
+  def respond_with(resource, _opts = {})
+    render json: {
+      status: {code: 200, message: 'Logged in successfully.' },
+      data: UserSerializer.new(resource).serializable_hash[:data][:attributes]
+    }, status: :ok
+  end
+
+  def respond_to_on_destroy
+    if current_user
+      render json: {
+        status: 200,
+        message: "logged out successfully"
```

Comment:
Write this as `Logged out successfully`

basically first character capital

Reply:
Changed to Logged